composer.json

```diff
@@ -9,13 +9,15 @@
             "EventSauce\\MessageOutbox\\DoctrineOutbox\\": "src/DoctrineOutbox",
             "EventSauce\\MessageRepository\\IlluminateMessageRepository\\": "src/IlluminateMessageRepository",
             "EventSauce\\MessageRepository\\DoctrineMessageRepository\\": "src/DoctrineMessageRepository",
-            "EventSauce\\MessageRepository\\TestTooling\\": "src/MessageRepositoryTestTooling"
+            "EventSauce\\MessageRepository\\TestTooling\\": "src/MessageRepositoryTestTooling",
+            "EventSauce\\MessageRepository\\DynamoDBMessageRepository\\": "src/DynamoDBMessageRepository"
```

Comment:
Question: do you prefer `DynamoDBMessageRepository` or `DynamoDbMessageRepository` ?
IMO it should be the first (`DynamoDBMessageRepository`)

Reply:
DynamoDBMessageRepository is preferred 👍